internal/handlers/sqlite/msg_explain.go

```diff
@@ -78,13 +79,18 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, lazyerrors.Error(err)
 	}
 
-	var qp backends.ExplainParams
-	if !h.DisableFilterPushdown {
-		qp.Filter = params.Filter
+	qp := backends.ExplainParams{
+		Filter: params.Filter,
+	}
+
+	sort := params.Sort
+
+	if params.Aggregate {
+		qp.Filter, sort = aggregations.GetPushdownQuery(params.StagesDocs)
```

Comment:
`sort` is not pushed down for aggregation but `params.Sort` is pushed down as far as I can see. Correct me if I'm wrong, `params.Sort` is nil for aggregation right? Could you add a test for it please and run it with `EnableSortPushdown` true please :hugs: 

```suggestion
		qp.Filter, params.Sort = aggregations.GetPushdownQuery(params.StagesDocs)
```